assets/layers/ticket_machine/ticket_machine.json

```diff
@@ -0,0 +1,235 @@
+{
+  "id": "ticket_machine",
+  "name": {
+    "en": "Ticket Machines"
+  },
+  "description": {
+    "en": "Find ticket machines for public transport tickets"
+  },
+  "source": {
+    "osmTags": {
+      "and": [
+        "amenity=vending_machine",
+        "vending=public_transport_tickets"
+      ]
+    }
+  },
+  "minzoom": 19,
+  "title": {
+    "render": {
+      "en": "Ticket Machine"
+    }
+  },
+  "tagRenderings": [
+    "images",
+    "level",
+    {
+      "id": "operator",
+      "question": {
+        "en": "Who is the operator of this ticket machine?"
+      },
+      "render": {
+        "en": "This ticket machine is operated by {operator}"
+      },
+      "freeform": {
+        "key": "operator",
+        "type": "string",
+        "placeholder": {
+          "en": "Name of the operator"
+        }
+      },
+      "mappings": [
+        {
+          "if": "operator=Nederlandse Spoorwegen",
+          "then": {
+            "en": "Dutch Railways (NS)",
+            "nl": "Nederlandse Spoorwegen (NS)"
+          },
+          "addExtraTags": [
+            "operator:wikidata=Q23076"
+          ],
+          "hideInAnswer": "_country!=nl"
+        }
+      ]
+    },
+    {
+      "id": "payment-options",
```

Comment:
A 'payment-options' already exists with the reusable 'questions'-json file; maybe this one is more appropriate there?

Reply:
I think for most items asking `payment:cash` is enough but some automated stuff might only have coin slots (or the other way around). I've moved it to questions.json, reusing the question and the original mappings (but hiding them)